census/routes/utils.js

```diff
@@ -22,7 +22,7 @@ var resolveProfile = function(profile, provider, done) {
   var obj = {
     id: uuid.v4(),
     anonymous: false,
-    emails: _.each(profile.emails, function(e, i, l) {
+    emails: _.forEach(profile.emails, function(e, i, l) {
```

Comment:
For future improvement: this `_.forEach` would be more straightforward as a [_.map()](https://lodash.com/docs/4.16.0#map).
Good to know fact: `_.each()` [is an alias in lodash for `_.forEach()`](https://lodash.com/docs/4.16.0#forEach).